skiplist/src/list.rs

```diff
@@ -223,7 +225,7 @@ impl<C: KeyComparator> Skiplist<C> {
         }
```

Comment:
Should we also change the logic of increasing height if we add `allow_concurrent_write` option?

Reply:
done